File: commons/src/main/java/ru/starksoft/commons/ColorUtils.java
package ru.starksoft.commons;

import androidx.annotation.ColorInt;
import androidx.annotation.NonNull;

public final class ColorUtils {

	private ColorUtils() {
		throw new UnsupportedOperationException();
	}

	@NonNull
	public static String hexColorFromInt(@ColorInt int color) {
		return String.format("#%06X", (0xFFFFFF & color));
	}
}

File: commons/src/main/java/ru/starksoft/commons/ColorUtils.kt
package ru.starksoft.commons

import androidx.annotation.ColorInt

object ColorUtils {

    @JvmStatic
    fun hexColorFromInt(
        @ColorInt
        color: Int
    ): String {
        return String.format("#%06X", 0xFFFFFF and color)
    }
}
